.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+# https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/keeping-your-dependencies-updated-automatically
+version: 2
+updates:
+  - package-ecosystem: "cargo"
```

Comment:
Should we set a reviewer as well or does it chose us automatically?

Reply:
I think it should still use our team because of the code owneship thing. We will see.